int64 xmin
int64 ymin
int64 xmax
int64 ymax
int16 id
int64 cen_x
int64 cen_y